napari/utils/_testsupport.py

```diff
@@ -134,16 +134,24 @@ def make_napari_viewer(
     monkeypatch,
     clean_themes,
 ):
-    """A fixture function that creates a napari viewer for use in testing.
+    """A pytest fixture function that creates a napari viewer for use in testing.
 
-    Use this fixture as a function in your tests:
+    This fixture will take care of creating a viewer and cleaning up at the end of the
+    test. When using this function, it is **not** necessary to use a `qtbot` fixture,
+    nor should you do any additional cleanup (such as using `qtbot.addWidget` or
+    calling `viewer.close()`) at the end of the test. Duplicate cleanup may cause
+    an error.
 
+    To use this fixture as a function in your tests:
+
+    def test_something_with_a_viewer(make_napari_viewer):
+        # `make_napari_viewer` takes any keyword arguments that napari.Viewer() takes
         viewer = make_napari_viewer()
```

Comment:
```suggestion
        def test_something_with_a_viewer(make_napari_viewer):
            # `make_napari_viewer` takes any keyword arguments that napari.Viewer() takes
            viewer = make_napari_viewer()
```

This extra indentation is needed so that this section will format as code rather than text.

Reply:
Thanks, bad copy pasta